src/canillita_newsitems_events_handler.erl

```diff
@@ -0,0 +1,35 @@
+-module(canillita_newsitems_events_handler).
+
+-behaviour(gen_event).
+
+-export([ init/1
+        , terminate/2
+        , handle_info/2
+        , handle_call/2
+        , code_change/3
+        , handle_event/2
+        ]).
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% gen_event functions.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+init([]) ->
+  {ok, []}.
+
+handle_info(_Info, State) ->
+  {ok, State}.
+
+handle_call(_Request, State) ->
+  {ok, not_implemented, State}.
+
+handle_event({canillita_newsitems, created, [Entity]}, State) ->
+  canillita_news_handler:notify({newsitem_created, Entity}),
```

Comment:
Why don't you just send the `Entity`, which (BTW) it's not a `#{}` as the spec of `canillita_news_handler:notify` says. It's a `canillita_newsitems:newsitem()`